src/rprblender/engine/engine.py

```diff
@@ -128,10 +128,13 @@ def setup_image_filter(self, settings):
                 self._disable_image_filter()
                 self._enable_image_filter(settings)
 
+            return True
+
         elif self.image_filter:
             self._disable_image_filter()
+            return True
 
-        return True
+        return False
```

Comment:
Logic is not changed in this function, please revert.

Reply:
It's dedicated to the Outliner window which causes updates when clicking on an object two times.